buildenv/jenkins/docker-slaves/ppc64le/centos7/Dockerfile

```diff
@@ -197,32 +180,18 @@ RUN cd /tmp \
   && cd .. \
   && rm -rf /tmp/make-4.1
 
-# Setup boot JDK for building Java 8 and 9
+# Setup boot JDK for building Java 8
 RUN ln -s java-1.8.0 /usr/lib/jvm/adoptojdk-java-80
```

Comment:
Will the Adopt jdk8 build now run on rhel/cent7? Should we grab it instead of using the default system one?

Reply:
I have not tried the adopt jdk8 build, but I can try it out if you want.